docs/model-relationships.md

```diff
@@ -5,21 +5,21 @@ title: Relationships
 
 A relationship in CML is a tuple composed of:
 
-The namespace of the type being referenced
-The type name of the type being referenced
-The identifier of the instance being referenced
+1. The namespace of the type being referenced
+2. The type name of the type being referenced
+3. The identifier of the instance being referenced
 
 Hence a relationship could be: `org.example.Vehicle#123456`
 
-This would be a relationship to the `Vehicle` type declared in the `org.example` namespace with the identifier `123456`.
+This would be a relationship to the `Vehicle` _type_ declared in the `org.example` _namespace_ with the _identifier_ `123456`.
 
 Relationships are unidirectional and deletes do not cascade, ie. removing the relationship has no impact on the thing that is being pointed to. Removing the thing being pointed to does not invalidate the relationship.
 
-Relationships must be resolved to retrieve an instance of the object being referenced. The act of resolution may result in null, if the object no longer exists or the information in the relationship is invalid. Resolution of relationships is outside of the scope of the Model Manager.
+Relationships must be resolved to retrieve an instance of the object being referenced. The act of resolution may result in null, if the object no longer exists or the information in the relationship is invalid. Resolution of relationships is outside of the scope of Concerto.
```

Comment:
I'm not sure if this was supposed to change or was meant to stay as Model Manager